scene/animation/animation_tree.cpp

```diff
@@ -542,6 +547,8 @@ void AnimationNode::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_filter_enabled", "enable"), &AnimationNode::set_filter_enabled);
 	ClassDB::bind_method(D_METHOD("is_filter_enabled"), &AnimationNode::is_filter_enabled);
 
+	ClassDB::bind_method(D_METHOD("get_process_animation_tree_id"), &AnimationNode::get_process_animation_tree_id);
```

Comment:
```suggestion
	ClassDB::bind_method(D_METHOD("get_processing_animation_tree_instance_id"), &AnimationNode::get_processing_animation_tree_instance_id);
```
More clarification and make consistent with some class property name.